pipe_satellite_data/process/sat_locations.py

```diff
@@ -28,6 +28,9 @@ def process(self, date_ts, norad_ids):
 
         # get TLEs for norad_ids for the date
         tles = fetch_TLE(self.st_auth, norad_ids, dt)
+
+        # https://www.space-track.org/documentation#api-basicSpaceDataDecay
+        tles = filter(lambda x: int(x["DECAYED"]) not in [1, 2], tles)
```

Comment:
Should we track somewhere LTEs that are decayed? could this information be needed for analysis, debugging or troubleshooting?

Reply:
@rdgfuentes good question. I can log them, to have track of them. Then we can decide what to do with it.